ELE_Patcher/Program.cs

```diff
@@ -180,7 +178,7 @@ private static void PatchModded(IPatcherState<ISkyrimMod, ISkyrimModGetter> stat
 		{
 			HashSet<ModKey> presentSupportedMods = new();
 			foreach (var mod in ModSupport.SupportedMods)
-				if (state.LoadOrder.HasMod(mod, true))
+				if (state.LoadOrder.ModExists(mod, true))
```

Comment:
Namings changed a bit as some calls just check the presence on the LO vs others check existence on disk as well